### DISCUSSION TITLE
How can I change op path?

### DISCUSSION THREAD

**Discussion by prenagha** [Ptujec/LaunchBar#30] [Q&A]
I install op via home-brew, as recommended on the 1password site.
This installs to `/opt/homebrew/bin/op`
The `default.js:27` script seems to hardcode it to `/usr/local/bin/op` and I don't see any code to override that with a preference.
What do you suggest?

**Comment by Ptujec** [comment]
I did a quick fix to except both paths (1.2.4). Maybe I'll add an option to pick a custom location later. Thanks for reporting!
